### PR TITLE
Build bootloader: fix waf on msvc.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: bootloader/wscript [hunk 1]
```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```
Comment (htgoebel) [inline bootloader/wscript]: 1. Is this related to msvc ot to `winrc` tool`?
2. Please add a comment, what this does
3. Why do we need to set ` WINRC_SRC_F` to an empty string?
4. These lines should go into the "configure" phase, where the CFLAGS are set - or if it is related to `winrc` into a new part of "tools flags" (after CFLAGS).

Comment (ghost) [reply bootloader/wscript]: This to set the correct compiler arguments when using MSVC. `WINRC_TGT_F` is used to determine the argument to use when linking objects and `-o` is the default (which MSVC doesn't understand). Let me see whether I can move this...

Edit: changed linker -> compiler.

Comment (ghost) [reply bootloader/wscript]: > Why do we need to set WINRC_SRC_F to an empty string?

Because with gcc, the include files are prefixed with `-i` whereas with visual studio they are not.

Comment (htgoebel) [reply bootloader/wscript]: Setting these flags is already done by waf, but only if WINRC is not set, see bottol of waflib/tools/winrc.py. So the correct solution is to not search for WINDRES if CC is msvc in line 399.

Comment (ghost) [reply bootloader/wscript]: That solution [doesn't appear to work][1].

[1]: https://ci.appveyor.com/project/htgoebel/pyinstaller/build/1.0.161

Comment (htgoebel) [reply bootloader/wscript]: This solution is unrelated to the error you linked, [see line 386][1].

[1]: https://ci.appveyor.com/project/htgoebel/pyinstaller/build/1.0.161#L386

Comment (ghost) [reply bootloader/wscript]: I don't think it is unrelated because [it does compile][1] when those variables are set in configure.

Edit: wrong link.

[1]: https://ci.appveyor.com/project/htgoebel/pyinstaller/build/1.0.155/job/39192i5owp9ttdlk

Comment (ghost) [reply bootloader/wscript]: Here is a [clean build][1] to show that it really does work on x64. Really I don't think too much time should be spent on this if two lines in configure can fix the issue.

[1]: https://ci.appveyor.com/project/htgoebel/pyinstaller/build/1.0.163

**Comment by ghost** [comment]
I am not currently sure, but I brought it up [here][1]. I will do some digging on that.


[1]: https://github.com/pyinstaller/pyinstaller/pull/2415#discussion_r98324528

File: bootloader/wscript (hunk 1 above)
Comment (htgoebel) [reply bootloader/wscript]: > I don't think too much time should be spent on this if two lines in configure can fix the issue.

The simplest solution is to use `--target-arch`
